server/api/rule_test.go

```diff
@@ -615,3 +616,92 @@ func (s *testRuleSuite) TestBatch(c *C) {
 		}
 	}
 }
+
+func (s *testRuleSuite) TestBundle(c *C) {
+	// GetAll
+	b1 := placement.GroupBundle{
+		ID: "pd",
+		Rules: []*placement.Rule{
+			{GroupID: "pd", ID: "default", Role: "voter", Count: 3},
+		},
+	}
+	var bundles []placement.GroupBundle
+	err := readJSON(testDialClient, s.urlPrefix+"/placement-rule", &bundles)
+	c.Assert(err, IsNil)
+	c.Assert(bundles, HasLen, 1)
```

Comment:
Looks like the test failed due to other inserted rules.

Reply:
Thanks, I'll take a look.